db/migrate/20160927230136_enable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class EnableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
```

Comment:
Right, migration *134 removes the fk, but the later migration *135 is already part of develop, so anyone that has pulled develop down will not run *134 by default, which puts them in the erroneous state.